CPPLogger/CPPLogger.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+
+#include "P7_Client.h"
+#include "P7_Trace.h"
+
+
+enum LogLevels
```

Comment:
It's better to use enum class instead of enum.

Reply:
I need to cast from my enum to lib enum inside one of my func. If i use enum class i couldn't do it.